openspec/changes/archive/2026-05-25-replace-laio-with-internal-support/design.md

```
@@ -0,0 +1,127 @@
+## Context
+
+`session-tmux` currently delegates window/pane layout to Laio via `pane_group_command`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)